KsApi/models/templates/RewardTemplates.swift

```diff
@@ -24,6 +25,7 @@ extension Reward {
 
   public static let noReward = Reward(
     backersCount: nil,
+    convertedMinimum: 0,
```

Comment:
I know this is just a template, but I think a `convertedMinimum` would never be 0 🤔 Maybe make this `1` so it's more realistic?

Reply:
I set `convertedMinimum` to  `0` so it matches the `minimum`. For this template, all properties are either `nil` or `0`.